configs/machines/ecmwf-atos.yaml

```
@@ -0,0 +1,125 @@
+# THE ECMWF Atos YAML config file
+
+name: ecmwf-atos
+
+accounting: true
+batch_system: "slurm"
+
+operating_system: "linux-centos"
+
+jobtype: compute
+sh_interpreter: "/usr/bin/bash"
+
+
+partitions:
+        compute:
+                name: mpp
+                cores_per_node: 128
+        pp:
+                name: smp
+                cores_per_node: 1
```

I have changed the name of the partitions here to match those of the documentation. You can accept this suggestion by clicking in the button below and committing.

```suggestion
        compute:
                name: compute
                cores_per_node: 128
        pp:
                name: gpil
                cores_per_node: 1
```